app/Http/Controllers/SubtitleController.php

```diff
@@ -120,7 +120,7 @@ public function store(Request $request)
         ]);
 
         if ($v->fails()) {
-            return redirect()->route('subtitles.create', ['torrent_id' => $request->input('torrent_id')])
+            return redirect()->route('subtitle.create', ['torrent_id' => $request->input('torrent_id')])
```

Comment:
This should remain plural. 
```
        // Subtitles System
        Route::group(['prefix' => 'subtitles'], function () {
            Route::name('subtitles.')->group(function () {
                Route::get('/', 'SubtitleController@index')->name('index');
                Route::get('/create/{torrent_id}', 'SubtitleController@create')->where('id', '[0-9]+')->name('create');
                Route::post('/store', 'SubtitleController@store')->name('store');
                Route::post('/{id}/update', 'SubtitleController@update')->name('update');
                Route::delete('/{id}/delete', 'SubtitleController@destroy')->name('destroy');
                Route::get('/{id}/download', 'SubtitleController@download')->name('download');
                Route::get('/filter', 'SubtitleController@faceted');
            });
        });
```

Routes are plural and view returns are not.

Reply:
Ah, I confused them the routes with views. I will change that back.

Reply:
Not a problem. Looks good otherwise.